.github/workflows/test.yml

```diff
@@ -1,19 +1,21 @@
 name: Test / Lint
 on:
   push:
+  pull_request:
```

Comment:
The `test-merge.yml` workflow was basically identical to this one. I thought might as well de-dupe. Happy to not do this if you prefer.